test/azure-pipelines.yml

```diff
@@ -1,7 +1,8 @@
 variables:
   ITKGitTag: v5.0.0
-  ITKPythonPost: '.post1'
+  ITKPythonGitTag: v5.0.0.post1
```

Comment:
After you get it working with this version, update to 5.0.1?

Reply:
:+1: yes :-)